CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-03-01
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-10 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on June 30th, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting last week, the week of Feb 22nd (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.2 for cluster 1.20.2 and above. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
+* Systemd-resolved will no longer be used in AKS Ubuntu 18.04 images starting on next week's release. This resolves past issues regarding private DNS with .local entries not working with [Kubernetes 1.18 and Ubuntu 18.04](https://github.com/Azure/AKS/issues/2052).
+
+### Release Notes
+
+* Features
+  * AKS Managed AAD now supports Just-in-Time Access. See more: <https://docs.microsoft.com/en-us/azure/aks/managed-aad#configure-just-in-time-cluster-access-with-azure-ad-and-aks>
+  * Application Gateway Ingress Controller (AGIC) AKS Add-On is now Generally Available [GA].
+  * Confidential computing nodes (DCSv2) AKS Add-on is now Generally Available [GA].
+  * HTTP Application Routing addon now Generally Available in Gov Cloud.
+  * Encrypted customer managed keys policy for AKS is now Generally Available [GA].
+  * Public IP per node capability in AKS is now Generally Avaiable [GA].
+  * Deploy WebLogic on Azure Kubernetes Service (AKS) using custom Docker images is now Generally Available [GA]. See more: <https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/oracle/weblogic-aks>.
+  * Persistent Volume monitoring & Reports tab in Container Insights is now Generally Available [GA]. See more:
+    * <https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-persistent-volumes>.
```

Comment:
In everyone of your links make sure that `en-us/` is deleted <del>en-us/</del>. We don't want docs pages to default to English for our users.